### ISSUE TITLE
Project Outline: Feedback

### ISSUE THREAD

**Issue by lindskelsey** [jcwhite99/liftoff-assignments#3]
I think your project sounds reasonable for the scope of 8 weeks, but also has the opportunity to become more complex if you'd like to work on it beyond 8 weeks.

I'd encourage you to formulate your MVP without including the ability to process payments. I would consider this a good "stretch goal". You can certainly use the PayPal API to process payments within your app, but this also adds the complexity of needing to use PayPal's Sandbox to replicate the payment process in order to test your app.

I think without having the payment feature, just being able to generate, store & share invoices solves a problem & would suffice as your MVP.

**Comment by jcwhite99** [comment]
I agree.  I am going to leave the pay now feature out for now and if I have time to add it in later then I will.  For now I'm just focused on exactly what you said, generating, storing and sharing invoices.  I think I am going to incorporate the gmail api for this.  I haven't had a chance to look at it too much but it seems promising.
